src/generated/rpg_terrain.h

```diff
@@ -89,9 +89,9 @@ namespace RPG {
 		int32_t special_lateral_party = 10;
 		int32_t special_lateral_enemies = 5;
 		int32_t grid_location = 0;
-		int32_t grid_a = 0;
-		int32_t grid_b = 0;
-		int32_t grid_c = 0;
+		int32_t grid_a = 120;
+		int32_t grid_b = 392;
+		int32_t grid_c = 16000;
```

Comment:
Intersting, this could fix battle issues we have in some RPG2k3 games! (all party members overlapping)

Reply:
Those were a huge pain to figure out. Basically I had to keep guessing and edit the ldb binary manually. Then read/write it with rm2k3 to see if the chunk would dissapear.

I had the same thought that maybe this would fix some bugs in Player. Any game with defaulted terrain would be all 0's and have overlapped actors.

Would you like me to cherry-pick this one out into a separate bug-fix PR so you can merge it for 5.4?

Reply:
Yeah, a cherry pick just for this would be great. thanks.